past_projects.md

```diff
@@ -0,0 +1,16 @@
+# Past Projects
+
+Welcome to the GMN Past Projects, a group of GMN projects by past participants. 
+
+The areas of project are not limited to: Contributing to the community, Running data analysis, Instructing students, and Carrying out projects. 
+
+## Here’s what our participants worked on
+
+### [Galaxy Tools](#)
```

Comment:
To what would that link?

Reply:
The link was supposed to redirect us to the different topics Gmentees worked on. The "Galaxy Tools" is like the topics group name, that is, all topics under it are related to tools. But since we are opting for "Past Gmentees" that would not be necessary again. 

Now the question is, how do we automate the grouping of Gmentees by topics?

Reply:
We could have an automated way to create this page.
For example, in OLS, all projects details (participants, mentor, description) are stored in YAML file. The content of the YAML file is imported in the template of the project page and is added to the final HTML page using loops.
We could think about having something similar, with one of the attributes being a topic. We could then in the template, parse the Gmentees and group them by topic.